CreeDictionary/DatabaseManager/xml_importer.py

```diff
@@ -520,22 +513,43 @@ def import_xmls(dir_name: Path, multi_processing: int = 1, verbose=True):
                         keyword_counter += 1
 
                 # now we create definition for all (possibly inflected) entries in xml that are forms of this lemma.
-                d_s_dicts = xml_lemma_to_definition_src_dicts.get(generated_wordform)
+                # d_s_dicts = xml_lemma_to_definition_src_dicts.get(generated_wordform)
```

Comment:
```suggestion
```

Reply:
@eddieantonio LOOLOOLOLL I did notice that line of undeleted commented out code when I was reviewing. And I was thinking "fine I'll just leave it there Eddie won't notice"